src/infection-model/seir.tsx

```diff
@@ -136,18 +170,28 @@ function simulateOneDay(inputs: SimulationInputs & SingleDayInputs) {
   ];
 }
 
+enum R0Cells {
+  low = 2.4,
+  moderate = 3,
+  high = 3.7,
+}
+
+enum R0Dorms {
+  low = 3,
+  moderate = 5,
+  high = 7,
+}
+
 function getCurveProjections(inputs: SimulationInputs & CurveProjectionInputs) {
   let {
+    ageGroupInitiallyInfected,
     ageGroupPopulations,
-    rateOfSpreadFactor,
+    facilityDormitoryPct,
+    facilityOccupancyPct,
     numDays,
-    initiallyInfected,
+    rateOfSpreadFactor,
   } = inputs;
 
-  // this number can't be zero;
-  // if it's missing, substitute a conservative assumption
-  initiallyInfected = initiallyInfected || 1;
```

Comment:
At first glance, I don't see this fallback happening for the age group'd initially infected numbers. Is that no longer needed, or am I not seeing where it happens?

Reply:
it shouldn't be needed anymore but there is still a gap in the overall data flow that makes sure this overall value is populated by _something_ ... so yeah you are not seeing where it happens because it's not happening yet 😎 . I think that is an issue for a separate PR though because it is more tightly coupled with the form UI and validation logic

Reply:
Do you think that logic is a launch blocker?

Reply:
yeah I have already discussed that with Serena also

Reply:
Cool, as long as another PR is on the way!